account_analytic_required/models/account.py

```diff
@@ -34,49 +34,31 @@ class AccountAccountType(models.Model):
         ),
     )
 
-
-class AccountAccount(models.Model):
-    _inherit = "account.account"
-
     def _get_analytic_policy(self):
         """Extension point to obtain analytic policy for an account"""
         self.ensure_one()
-        return self.user_type_id.with_company(
-            self.company_id.id
-        ).property_analytic_policy
+        return self.analytic_policy
 
 
 class AccountMove(models.Model):
     _inherit = "account.move"
 
     def _post(self, soft=True):
-        res = super()._post(soft=soft)
+        res = super(AccountMove, self)._post(soft=soft)
```

Comment:
you don't change `self`, why do we need this change?

Reply:
i just want to make it explicitly, anyway, i have updated to make no change here